roles/sideload_kernel/tasks/main.yml

```diff
@@ -0,0 +1,117 @@
+---
+- name: Validate required variables
+  ansible.builtin.assert:
+    that:
+      - sideload_kernel_uri | string
+
+- name: Install python dependencies
```

Comment:
don't install python packages via pip, install the required RPM instead, or, if there is no RPM, at least install this in a python virtual environment

Reply:
This would also be a good place to ensure `jq` is installed

Reply:
I was curious about this, and just generally how we are supposed to handle dependencies of these ansible roles...

I see other roles that also use community.kubernetes.k8s without any explicit installation of the required python libaries.  Is there a better way to do this?

Reply:
I tried a version that uses ansible.builtin.package to install the RPM, with a fallback to pip... but that requires becoming root on the host, which seems a bit heavy considering nothing else in this role requires it...

Maybe I should remove this and rely on people reading the dependencies section of the README?

Reply:
you can have a validate section at the top of main.yml (or include a validate.yml or whatever) to make sure you've got all the things you need to run the sideload script and bail early if not with a useful message.